hydradx/notebooks/Omnipool/TestLpFeeDeposit.ipynb

```diff
@@ -0,0 +1,445 @@
+{
```

Comment:
When I try to run this notebook I get
'TypeError: __init__() got an unexpected keyword argument 'lp_lrna_share''

Reply:
I should probably just delete the notebook, since we decided against using that mechanism